src/sentry/integrations/slack/message_builder/notifications.py

```diff
@@ -1,98 +0,0 @@
-from __future__ import annotations
```

Comment:
Why delete this file and make it 3? This PR is difficult to review because you have to mentally ignore the "noise" and try to find the actual changes which take up the minority of the diff from what I can tell. Can you remove the unrelated things and just have a PR with the changes?

Reply:
Yes, that would make reviewing much easier